### PR TITLE
Fix #2341: Replace liqd/django-autoslug dependency to upstream 

### LINKED ISSUE #2341: move back to upstream django-autoslug
As our PRs have been merged we can stop using our fork and move back to the official package of django-autoslug

### PR DESCRIPTION
**Pull request by DubeySandeep**
## Overview

This PR fixes #2341 by replacing the [forked/edited django-autoslug](https://github.com/liqd/django-autoslug) (introduced via #2188) with its original upstream.

This is possible now as the changes made on the forked repo are [merged and released upstream](https://github.com/justinmayer/django-autoslug/pull/79#issuecomment-1494295682).

## Manual testing

**Able to install the latest upstream django-autoslug via pip**
image

**Able to run the server with the latest version of  django-autoslug**
image

### CODE REVIEW

File: requirements/base.txt
Comment (DubeySandeep) [inline requirements/base.txt:19]: Looks like a4-core also uses this dep., I'll initiate a PR there as well so that we can archive/delete (if needed) [liqd/django-autoslug](https://github.com/liqd/django-autoslug) once bother the PRs are merged.

Comment (DubeySandeep) [reply requirements/base.txt:19]: Done! (Initiated a PR https://github.com/liqd/adhocracy4/pull/1413)

**Comment by goapunk** [comment]
Hey @DubeySandeep, thanks for your PRs and contributions! We are currently discussing internally whether we should have a CLA for external contributors (we probably shouldn't have one... :) ) but as this is a team decision it will still take us some time and I would like to wait until then before we merge this, sorry!

**Review by goapunk — APPROVED** [review]

**Comment by DubeySandeep** [comment]
@goapunk Thanks for the review and the heads-up! I think it's a good idea to have a CLA (assuming t will be a simple form (max 3 inputs)), I'm fine with signing the CLA, let me know once you have an update! :)

Also, it looks like we are using a secret ([secret.COV](https://github.com/liqd/adhocracy-plus/blob/f6837eecea9de72327c4351e65bcd7dfae154c76/.github/workflows/django.yml#L81)) which is not permissible to forked PRs build ([ref](https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/#improvements-for-public-repository-forks)), resulting into [empty string value for the token](https://github.com/liqd/adhocracy-plus/actions/runs/5093158170/jobs/9203351856?pr=2446#step:11:11) and blocking my PR build to complete the `Coveralls` check. 

Possible solutions: 
- Use `secrets.GITHUB_TOKEN` similar to [line 92](https://github.com/liqd/adhocracy-plus/blob/f6837eecea9de72327c4351e65bcd7dfae154c76/.github/workflows/django.yml#L92) (**Note:** I don't have context for the usage of COV secret and whether it can be replaced by `secrets.GITHUB_TOKEN`)
- Use `pull_request_target` event as suggested in [the doc](https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks#about-workflow-runs-from-public-forks).

**Comment by goapunk** [comment]
Hi @DubeySandeep, sorry for the late reply and thanks again for your contribution. We should have something ready by the end of the week. We'll also look into fixing the CI/Coveralls for external PRs, thanks for reporting the issue and the suggestions!

**Comment by goapunk** [comment]
superseded by https://github.com/liqd/adhocracy-plus/pull/2710

**Pull request closed without merging**
